pyDeltaRCM/command_line.py

```diff
@@ -0,0 +1,34 @@
+import os
+import argparse
+
+import pyDeltaRCM as deltaModule
```

Comment:
why? can't you get pyDeltaRCM version from `v = pydeltarcm.shared_tools._get_version()` ([here](https://github.com/DeltaRCM/pyDeltaRCM_WMT/blob/fd0be0d091af7056a4004cace13c476ca5163b3d/pyDeltaRCM/shared_tools.py#L41)) or, even better, from `pyDeltaRCM.__version__` ([here](https://github.com/DeltaRCM/pyDeltaRCM_WMT/blob/fd0be0d091af7056a4004cace13c476ca5163b3d/pyDeltaRCM/__init__.py#L7)) I think this would be preferred to declaring an import convention we don't use anywhere else.

Reply:
Agreed. as you mention in your next comment, the issue was that the namespaces conflicted, and this was the solution I came up with while messing around. I'll do what you say below I think.